src/main/scala/com/bwsw/kafka/reader/Consumer.scala

```diff
@@ -60,6 +60,7 @@ class Consumer[K,V](settings: Consumer.Settings) {
     * @param topicPartitionInfoList entity which contains topics, partitions and offsets.
     */
   def assignWithOffsets(topicPartitionInfoList: TopicPartitionInfoList): Unit = {
```

Comment:
add a log before "consumer.assign" and a log before seek circle

Reply:
ok